benches/algos.rs

```diff
@@ -70,6 +70,17 @@ fn corner_to_corner_bfs(c: &mut Criterion) {
     });
 }
 
+fn corner_to_corner_bfs_bidirectional(c: &mut Criterion) {
```

Comment:
This actually runs slower than `corner_to_corner_bfs`. I believe the problem is too small for the bidirectional search to give any advantage.

Reply:
Could you add a larger example where this makes a difference? Not too large to prevent the CI from running smootly, but large enough to show bidirectional BFS advantages. 

Note that you can craft the problem as you wish, the idea is just to show that in at least one situation bidirectional BFS is a clear win.

Reply:
I've added an example now - please have a look an see what you think 🙂